### PR TITLE
Improve instructions for fixing invalid metadata

### PR DESCRIPTION
**Pull request by Eric-Arellano**
Closes https://github.com/Qiskit/documentation/issues/200.

Note that we still want to improve this metadata checker to also start checking some quality things, like that it's <160 characters. But this is a good first pass. See https://github.com/Qiskit/documentation/issues/327.

```
      Invalid Jupyter notebook metadata. Every .ipynb file needs to 
      set 'title' and 'description' in the file metadata. You need to
      manually add this metadata. 
      
      For example, if using VSCode, open up the file with the "Open With..." option and then "Text Editor".
      
      Once the file is open in text-mode, scroll down to the bottom of the file for the top-level key "metadata". Be careful that this is the metada for the entire file
      and not a single code block. You should see in the "metadata" section other
      entries like "language_info" and "nbconvert_exporter".

      Finally, add new keys in the "metadata" section for "title" and "description". The
      "title" should be the page title; the "description" should describe the page
      in <160 characters, ideally using some keywords. The description is what
      shows up on Google results. See https://github.com/Qiskit/documentation/issues/131 for some tips. 

      For example:

      "metadata": {
        "description": "Get started using Qiskit with IBM Quantum hardware in this Hello World example",
        "title": "Hello world",
        "celltoolbar": "Raw Cell Format",
        "kernelspec": { ...
      }

      Please fix these files: docs/start/hello-world.ipynb
```

```
      Invalid markdown file metadata. Every .md and .mdx file should start with a metadata block like this:

      ---
      title: Representing quantum computers
      description: Learn about coupling maps, basis gates and backend errors for transpiling
      ---

      The title should be the page title; the description should describe the page
      in <160 characters, ideally using some keywords. The description is what
      shows up on Google results. See https://github.com/Qiskit/documentation/issues/131 for some tips.

      Please fix these files: docs/start/setup-channel.mdx
    ```

### CODE REVIEW

File: scripts/commands/checkMetadata.ts [hunk 1]
```diff
@@ -76,18 +76,42 @@ const main = async (): Promise<void> => {
...
   if (notebookErrors.length > 0) {
     console.error(`
       Invalid Jupyter notebook metadata. Every .ipynb file needs to 
-      set 'title' and 'description' in the file metadata.
+      set 'title' and 'description' in the file metadata. You need to
+      manually add this metadata. 
+      
+      For example, if using VSCode, open up the file with the "Open With..." option and then "Text Editor".
+      
+      Once the file is open in text-mode, scroll down to the bottom of the file for the top-level key "metadata". Be careful that this is the metada for the entire file
```
Comment (arnaucasau) [inline scripts/commands/checkMetadata.ts:98]: Nit:
```suggestion
      Once the file is open in text-mode, scroll down to the bottom of the file for the top-level key "metadata". Be careful that this is the metadata for the entire file
```

**Review by arnaucasau — APPROVED** [review]
I think it's all very clear and helpful. I tried to follow the instructions as someone not familiar with the process, and I was able to fix a file I created as a test without any problem!

File: scripts/commands/checkMetadata.ts [hunk 2]
```diff
@@ -76,18 +76,42 @@ const main = async (): Promise<void> => {
...
+      manually add this metadata. 
+      
+      For example, if using VSCode, open up the file with the "Open With..." option and then "Text Editor".
+      
+      Once the file is open in text-mode, scroll down to the bottom of the file for the top-level key "metadata". Be careful that this is the metadata for the entire file
+      and not a single code block. You should see in the "metadata" section other
+      entries like "language_info" and "nbconvert_exporter".
+
+      Finally, add new keys in the "metadata" section for "title" and "description". The
+      "title" should be the page title; the "description" should describe the page
```
Comment (abbycross) [inline scripts/commands/checkMetadata.ts:103]: Without breaking this page by making a line too long, I'd suggest adding the following: 
```suggestion
      "title" should be the page title, and will appear in browser tabs; the "description" should describe the page
```

File: scripts/commands/checkMetadata.ts [hunk 3]
```diff
@@ -76,18 +76,42 @@ const main = async (): Promise<void> => {
       Invalid markdown file metadata. Every .md and .mdx file should start with a metadata block like this:
 
       ---
-      title: OpenQASM 3 feature table
-      description: A list of the OpenQASM 3 language features
+      title: Representing quantum computers
+      description: Learn about coupling maps, basis gates and backend errors for transpiling
```
Comment (abbycross) [inline scripts/commands/checkMetadata.ts:80]:
```suggestion
      description: Learn about coupling maps, basis gates, and backend errors for transpiling
```
(teeny tweak: Oxford comma :))

**Review by kevinsung — APPROVED** [review]

File: scripts/commands/checkMetadata.ts (hunk 2 above)
Comment (abbycross) [reply scripts/commands/checkMetadata.ts:103]: I believe this is the same for .mdx/.md metadata, that the browser tabs reflect the title from the metadata

**Review by abbycross — APPROVED** [review]
👍 thanks!

**Pull request merged**